### PR TITLE
LEGO: Pull request from users/loc/juno/hb_bcf03584-5f49-46b3-b9d9-20b47d2b6ca7_20230310203353728 to main

### PR DESCRIPTION
**Pull request by csigs**
LEGO: Pull request from users/loc/juno/hb_bcf03584-5f49-46b3-b9d9-20b47d2b6ca7_20230310203353728 to main with localized lcls

### CODE REVIEW

File: src/loc/lcl/PLK/Axe.Windows.Rules.lcl [hunk 1]
```diff
@@ -2180,10 +2180,13 @@
...
           <Tgt Cat="Text" Stat="Loc" Orig="New">
-            <Val><![CDATA[Określ właściwość Name automatyzacji interfejsu użytkownika dla elementu, który:]D;]A; · Zwięźle identyfikuje element, ORAZ]D;]A; · Nie zawiera nazwy klasy elementu (np. Microsoft.*.* lub Windows.*.*).]]></Val>
+            <Val><![CDATA[Sprawdź właściwość Nazwa automatyzacji interfejsu użytkownika tego elementu:]D;]A; · Jeśli właściwość Nazwa zawiera ciąg „Microsoft” lub „Windows” i jest ściśle powiązana z tekstem wyświetlanym na ekranie (na przykład odwołaniem do elementu „Microsoft.com”), możesz bezpiecznie zignorować ten problem.]D;]A; · Jeśli właściwość Nazwa nie jest ściśle powiązana z tekstem wyświetlanym na ekranie (i nie udostępnia użytkownikom końcowym istotnych informacji), zaktualizuj właściwość Nazwa na zwięzłą etykietę skierowaną do użytkownika, która jest ściśle powiązana z tekstem wyświetlanym na ekranie.]]></Val>
```
Comment (codeofdusk) [inline src/loc/lcl/PLK/Axe.Windows.Rules.lcl:2185]:
```suggestion
            <Val><![CDATA[Sprawdź właściwość Name automatyzacji interfejsu użytkownika tego elementu:]D;]A; · Jeśli właściwość Nazwa zawiera ciąg „Microsoft” lub „Windows” i jest ściśle powiązana z tekstem wyświetlanym na ekranie (na przykład odwołaniem do elementu „Microsoft.com”), możesz bezpiecznie zignorować ten problem.]D;]A; · Jeśli właściwość Nazwa nie jest ściśle powiązana z tekstem wyświetlanym na ekranie (i nie udostępnia użytkownikom końcowym istotnych informacji), zaktualizuj właściwość Nazwa na zwięzłą etykietę skierowaną do użytkownika, która jest ściśle powiązana z tekstem wyświetlanym na ekranie.]]></Val>
```

File: src/loc/lcl/KOR/Axe.Windows.Rules.lcl [hunk 2]
```diff
@@ -2180,10 +2180,13 @@
...
+          <Val><![CDATA[Examine the UI Automation Name property of this element:]D;]A; · If the Name property contains "Microsoft" or "Windows" and is closely related to the on-screen text (for example, a reference to "Microsoft.com"), you may safely ignore this issue.]D;]A; · If the Name property is not closely related to the on-screen text (and provides no meaningful information to end-users), update the Name to a concise, user-facing label that is closely related to the on-screen text.]]></Val>
           <Tgt Cat="Text" Stat="Loc" Orig="New">
-            <Val><![CDATA[다음 요소의 UI 자동화 Name 속성을 제공합니다.]D;]A; · AND 요소를 간결하게 식별합니다.]D;]A; · 요소의 클래스 이름(예: Microsoft.*.* 또는 Windows.*.*)을 포함하지 않습니다.]]></Val>
+            <Val><![CDATA[이 요소의 UI 자동화 이름 속성을 검사합니다.]D;]A; · 이름 속성이 "Microsoft" 또는 "Windows"를 포함하며 화면 텍스트(예: "Microsoft.com"에 대한 참조)와 긴밀하게 관련된 경우 이 문제를 무시해도 됩니다.]D;]A; · 이름 속성이 화면 텍스트와 긴밀하게 관련되지 않고 최종 사용자에게 의미 있는 정보를 제공하지 않는 경우 화면 텍스트와 긴밀하게 관련된 간결하고 사용자 관련 레이블로 이름을 업데이트합니다.]]></Val>
```
Comment (codeofdusk) [inline src/loc/lcl/KOR/Axe.Windows.Rules.lcl:2185]: Please use the word `Name` (written in roman letters) to refer to the UI Automation property.

**Review by codeofdusk — CHANGES_REQUESTED** [review]
Similar requests as #890.

**Comment by codeofdusk** [comment]
FYI @csigs

**Comment by DaveTryon** [comment]
Closing as requested by @csigs

**Pull request closed without merging**